Apsitvarkom.DataAccess/GoogleGeocoder.cs

```diff
@@ -23,7 +23,7 @@ public GoogleGeocoder(HttpClient httpClient, IApiKeyProvider apiKeyProvider)
 
     public async Task<string?> ReverseGeocodeAsync(Coordinates coordinates)
     {
-        var jsonStream = await _httpClient.GetStreamAsync($"json?latlng={coordinates.Latitude},{coordinates.Longitude}&language=lt&result_type=political&key={_apiKey}");
+        var jsonStream = await _httpClient.GetStreamAsync($"json?latlng={coordinates.Latitude},{coordinates.Longitude}&language=lt&result_type=point_of_interest&key={_apiKey}");
```

Comment:
As we've discussed privately, I don't see the need to have `&result_type=point_of_interest` here, since if a location won't be associated with any point of interest, then it will return no results (although it could have give us just the street address). We will be picking out just the first result and that should be enough for us